crates/primitives/src/lib.rs

```diff
@@ -12,7 +12,6 @@ mod constants;
 pub mod db;
 pub mod env;
 
-#[cfg(feature = "c-kzg")]
 pub mod kzg;
```

Comment:
```suggestion
#[cfg(any(feature = "c-kzg, feature = "kzg-rs"))]
pub mod kzg;
```

Have kzg only if there is any of those features.